skins/cat17/src/scripts/supporters.js

```diff
@@ -0,0 +1,38 @@
+(function( $ ) {
+	$( document ).ready( function() {
+		initSupportersTable();
+		attachEvents();
+	} );
+
+	function initSupportersTable() {
+		$( '.donors tbody tr' ).each( function() {
+			if( getComment( this ) ) {
+				$( this ).addClass( 'commented' );
+			}
+		} );
+	}
+
+	function attachEvents() {
+		var commentedRows = $( '.commented' );
+		commentedRows.click( function() {
+			if( $( this ).hasClass( 'active' ) ) {
+				hideComment.call( $( this ) )
```

Comment:
By using `.call`, you set the `this` value inside the `showComment` and `hideComment` functions to `$(this)` (from `attachEvents`). Inside the functions you do a `$(this)` which is a no-op, as jQuery will just return the object. Why not use `$( this )` as a `tableRow` parameter?